setup.py

```diff
@@ -0,0 +1,7 @@
+from setuptools import setup
+
+setup(name='collect_social',
+      packages=['collect_social'],
+      zip_safe=False,
+      license='MIT',
+      )
```

Comment:
In my mind, the advantage of a setup.py is that a module can be installed into the python path, enabling any program to call the module from anywhere on the filesystem.

For example, we could create an `examples` subdirectory with runnable examples. Without installing the module, this would require manually updating the python path or copying `collect_social` into the directory.

I'm not aware of any cons, though adding a setup.py might be more appropriate for a separate PR.